examples/GPS_SoftwareSerial_LOCUS_Start/GPS_SoftwareSerial_LOCUS_Start.ino

```diff
@@ -7,43 +7,37 @@
 // Tested and works great with the Adafruit Ultimate GPS module
 // using MTK33x9 chipset
 //    ------> http://www.adafruit.com/products/746
-// Pick one up today at the Adafruit electronics shop 
+// Pick one up today at the Adafruit electronics shop
 // and help support open source hardware & software! -ada
-//This code is intended for use with Arduino Leonardo and other ATmega32U4-based Arduinos
+
+// Only for 328 and ESP8266
+#if defined(__AVR_ATmega328P__) || defined(ESP8266)
 
 #include <Adafruit_GPS.h>
 #include <SoftwareSerial.h>
 
 // Connect the GPS Power pin to 5V
 // Connect the GPS Ground pin to ground
-// If using software serial (sketch example default):
-//   Connect the GPS TX (transmit) pin to Digital 8
-//   Connect the GPS RX (receive) pin to Digital 7
-// If using hardware serial:
-//   Connect the GPS TX (transmit) pin to Arduino RX1 (Digital 0)
-//   Connect the GPS RX (receive) pin to matching TX1 (Digital 1)
-
-// If using software serial, keep these lines enabled
-// (you can change the pin numbers to match your wiring):
-//SoftwareSerial mySerial(8, 7);
-//Adafruit_GPS GPS(&mySerial);
-
-// If using hardware serial, comment
-// out the above two lines and enable these two lines instead:
-Adafruit_GPS GPS(&Serial1);
-HardwareSerial mySerial = Serial1;
+// Connect the GPS TX (transmit) pin to Digital 8
+// Connect the GPS RX (receive) pin to Digital 7
+
+// you can change the pin numbers to match your wiring:
+SoftwareSerial mySerial(8, 7);
+Adafruit_GPS GPS(&mySerial);
 
 // Set GPSECHO to 'false' to turn off echoing the GPS data to the Serial console
 // Set to 'true' if you want to debug and listen to the raw GPS sentences
 #define GPSECHO  true
 
 // this keeps track of whether we're using the interrupt
 // off by default!
+#ifndef ESP8266 // Sadly not on ESP8266
 boolean usingInterrupt = false;
 void useInterrupt(boolean); // Func prototype keeps Arduino 0023 happy
```

Comment:
try removing

Reply:
8266 doesn't have TIMER0_COMPA_vect so the interrupt part fails when building for those.

Reply:
woops it marked the wrong line, or i was clumsy
i meant `void useInterrupt(boolean); // Func prototype keeps Arduino 0023 happy`